<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFaxDialog</class>
 <widget class="QDialog" name="SendFaxDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/res/icons/fax-ico.png</normaloff>:/icons/res/icons/fax-ico.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="1" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>217</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>231</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="loadingLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Loading...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>231</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>217</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resource.qrc">:/icons/res/icons/fax-ico.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="previewBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Preview</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="prevPageButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resource.qrc">
                    <normaloff>:/icons/res/icons/prev-ico.png</normaloff>:/icons/res/icons/prev-ico.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="pagesLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="nextPageButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="resource.qrc">
                    <normaloff>:/icons/res/icons/next-ico.png</normaloff>:/icons/res/icons/next-ico.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="0">
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>To:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="toAddressbookButton">
                <property name="text">
                 <string>Addressbook</string>
                </property>
                <property name="icon">
                 <iconset resource="resource.qrc">
                  <normaloff>:/icons/res/icons/addressbook-ico.png</normaloff>:/icons/res/icons/addressbook-ico.png</iconset>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Sender name:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Sender number:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="fromNo"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="toNo"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="fromName"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Price:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="priceLabel">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sendButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Send fax</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/icons/res/icons/fax-ico.png</normaloff>:/icons/res/icons/fax-ico.png</iconset>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fromName</tabstop>
  <tabstop>fromNo</tabstop>
  <tabstop>toNo</tabstop>
  <tabstop>toAddressbookButton</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>prevPageButton</tabstop>
  <tabstop>nextPageButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>prevPageButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>prevPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPageButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>nextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toAddressbookButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>toAddressbook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toNo</sender>
   <signal>editingFinished()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>toNoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toNo</sender>
   <signal>textEdited(QString)</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>toNoEdited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFaxDialog</receiver>
   <slot>sendFax()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>prevPage()</slot>
  <slot>nextPage()</slot>
  <slot>toAddressbook()</slot>
  <slot>toNoChanged()</slot>
  <slot>toNoEdited()</slot>
  <slot>sendFax()</slot>
 </slots>
</ui>
